configs/template_config.yaml

```diff
@@ -81,6 +81,13 @@ tokenized_dataset_path: "<YOUR_PATH_HERE>/data/tokenized_dataset/c4"
 # TensorBoard Path (str): Path for where TensorBoard files go, optional
 tboard_path: "<YOUR_PATH_HERE>/data/runs"
 
+# Split Dataset Boolean (T/F): Decide if you want to train your model on a smaller portion of C4
+split_dataset: true
+# Specify Percentage of Dataset (int): Decide what percentage of C4 you want to train on
+split_dataset_percentage: 1
```

Comment:
If it's not too much of a pain to switch, can this be changed to a float? The other paramaters that represent percentages are entered as floats (e.g., `val_check_interval`, `activation_dropout`, `splits`, etc.)

Reply:
I even think actually we could collapse both 'split_dataset' and `train_model_data_path` into `split_dataset_percentage` where default is `~`, `1`, or `1.0`, but all evaluates to the using the whole dataset. If `split_dataset_percentage != 1`, then we use the portion_data instead of train for the path and we use the decimal value as the split. The advantage being less paramters in the yaml.